odoo/addons/test_new_api/tests/test_onchange.py

```diff
@@ -539,6 +539,12 @@ def mock_read(self, fields=None, load='_classic_read'):
 
 class TestComputeOnchange(common.TransactionCase):
 
+    def test_01(self):
```

Comment:
Suggested method name: `test_onchange_once`, and please move that method right after `test_onchange` below.